src/containers/signup/Signup.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import './signup.less';
+import {Button, Col, Form, Input, Row, Typography} from 'antd';
+import { signup } from '../../auth/authAPI';
+import {Link} from 'react-router-dom';
+/*const {  } = Typography;*/
+
+const rSpan = 8;
+const passSpan = 17;
+
+const Signup: React.FC = () => {
+    const onFinish = (values: any) => {
+        signup({ email: values.username, password: values.password })
+    };
+
+    return (
+        <>
+            <div className="body-content-container">
+                <Row>
+                    <Col span={10} className="input-container">
+                        <h1>Sign Up</h1>
+                        <hr/>
+                        <Form
+                            name="basic"
+                            initialValues={{ remember: true }}
+                            onFinish={onFinish}
+                        >
+                            <Row>
+                                <Col className="leftInput" span={rSpan}>
+                                    <Form.Item
+                                        name="fname"
+                                        rules={[{ required: true, message: 'Please input your first name!' }]}
+                                    >
+                                        <Input placeholder="First Name"/>
+                                    </Form.Item>
+                                </Col>
+                                <Col span={rSpan}>
+                                    <Form.Item
+                                        name="lname"
+                                        rules={[{ required: true, message: 'Please input your last name!' }]}
+                                    >
+                                        <Input placeholder="Last Name"/>
+                                    </Form.Item>
+                                </Col>
+                            </Row>
+                            <Row>
+                                <Col className="leftInput" span={rSpan}>
+                                    <Form.Item
+                                        name="email"
+                                        rules={[{ required: true, message: 'Please input your email!' }]}
+                                    >
+                                        <Input placeholder="Email"/>
+                                    </Form.Item>
+                                </Col>
+                                <Col span={rSpan}>
+                                    <Form.Item
+                                        name="cemail"
+                                        rules={[{ required: true, message: 'Please confirm your email!' }]}
+                                    >
+                                        <Input placeholder="Confirm Email"/>
```

Comment:
We don't confirm email addresses. That's a mistake in the design sorry, but can you take it out and just make it a long input field like the password ones below?

Reply:
Fixed!
<img height="150" src="https://user-images.githubusercontent.com/22990100/97758528-6f253e00-1ad5-11eb-8245-2836e78dc69b.png">